portality/tasks/find_discontinued_soon.py

```diff
@@ -0,0 +1,137 @@
+import datetime
+import csv
+import json
+
+from portality.core import app
+from portality.bll import DOAJ
+from portality import models,app_email
+
+from portality.tasks.redis_huey import main_queue
+
+from portality.background import BackgroundTask, BackgroundSummary, BackgroundApi
+from portality.tasks.helpers import background_helper
+from portality.ui.messages import Messages
+from portality import constants
+
+
+def _date():
+    return (datetime.datetime.today() + datetime.timedelta(days=app.config.get('DISCONTINUED_DATE_DELTA', 1))).strftime(
+        '%Y-%m-%d')
+class DiscontinuedSoonQuery:
+    @classmethod
+    def query(cls):
+        return {
+            "query": {
+                "bool": {
+                    "filter": {
+                        "bool" : {
+                            "must": [
+                                {"term" : {"bibjson.discontinued_date": _date()}},
+                                {"term" : {"admin.in_doaj":True}}
+                            ]
+                        }
+                    }
+                }
+            }
+        }
```

Comment:
This is a bit of a strange construction, I would refactor this to follow our typical style in Query wrappers, and add a bit of generality.  In addition, use of the `portality.lib.dates` library instead of direct calls to `datetime`

```
class DiscontinuedSoonQuery:
    def __init__(self, time_delta=None):
        self._delta = time_delta if time_delta is not None else 0
        self._date = ... # calculate it as per _date (but using the dates library)
    
    def query(self):
        return {"query": { ... }}
```